MCC linter error

```
❯ ruleguard -c=0 -rules ../semgrep-go/ruleguard.rules.go ./...
/Users/andig/htdocs/evcc/charger/mcc.go:348:9: unconvert: unnecessary conversion (ruleguard.rules.go:31)
348		return time.Duration(currentSession.Duration * time.Second), nil
```

If `currentSession.Duration` really is a duration it should not need conversion. I assume it's rather an `int64` by nature?

--- a/charger/mcc.go
+++ b/charger/mcc.go
@@ -32,7 +32,7 @@ type MCCTokenResponse struct {
 
 // MCCCurrentSession is the apiCurrentSession response
 type MCCCurrentSession struct {
-	Duration     time.Duration
+	Duration     int64
 	EnergySumKwh float64
 }
 
@@ -345,7 +345,7 @@ func (mcc *MobileConnect) ChargingTime() (time.Duration, error) {
 		return 0, err
 	}
 
-	return time.Duration(currentSession.Duration * time.Second), nil
+	return time.Duration(currentSession.Duration) * time.Second, nil
 }
 
 var _ api.MeterCurrent = (*MobileConnect)(nil)

